teuthology/suite/__init__.py

```diff
@@ -112,10 +112,7 @@ def main(args):
         teuthology.log.setLevel(logging.DEBUG)
 
     if not conf.machine_type or conf.machine_type == 'None':
-        schedule_fail("Must specify a machine_type")
-    elif 'multi' in conf.machine_type:
-        schedule_fail("'multi' is not a valid machine_type. " +
-                      "Maybe you want 'plana,mira,burnupi' or similar")
+        conf.machine_type = config.default_machine_type
```

Comment:
I guess we need to revert this as well. Because we still need to inform the user if the machine type is not set.

Reply:
@kyr good point, I added a secondary check for it: https://github.com/ceph/teuthology/pull/1625/files#diff-8743f0bc16e084716b1065401e43959ca8f5a445a45a0299058673edfbde084dR115